test/sequential/market.nonTransferable.multiswap.test.ts

```diff
@@ -0,0 +1,294 @@
+import { jest } from "@jest/globals";
```

Comment:
group here! a new one, cause we require some specific build.

Reply:
Done